remote_vnc_backend_template.sh

```diff
@@ -18,8 +18,13 @@ create_vnc_password() {
 
 
 create_vncserver_unit_file () {
-	cp /{lib,etc}/systemd/system/vncserver@.service
-	sed -i "s|<USER>|<USERNAME>|g" /etc/systemd/system/vncserver@.service
+	if [ -f /lib/systemd/system/vncserver@.service ] ; then
+		cp /{lib,etc}/systemd/system/vncserver@.service
+		sed -i "s|<USER>|<USERNAME>|g" /etc/systemd/system/vncserver@.service
+	else
```

Comment:
Can we have one more `elif [ -f /usr/lib/systemd/system/vncserver@.service ] ...` here just to cover all possible locations? Sed for this case should be the same as for the `/lib/systemd/system/vncserver@.service`.